oasislmf/pytools/data_layer/conversions/correlations.py

```diff
@@ -1,12 +1,14 @@
 """
 This file defines the functions that convert correlations.
 """
-import os
+import sys
+from pathlib import Path
 
-from oasislmf.pytools.data_layer.oasis_files.correlations import CorrelationsData
+from oasislmf.pytools.data_layer.oasis_files.correlations import \
+    CorrelationsData
```

Comment:
again not sure why you have this extra line I would remove it

Reply:
done